File: Covid.Business/Model/StatesData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Covid.Business.Model
{
    public class StatesData
    {
        public string StateAbbreviation { get; set; }

        public string StateName { get; set; }

        public List<DaysData> AllDaysData { get; set; }

        public DateTime?[] Dates { get; set; }

        public double? MostRecentTotalCases { get; set; }

        public double? ProjectedTotalCases { get; set; }

        public double? MostRecentNewCases { get; set; }

        public double? MostRecentTotalDeaths { get; set; }

        public double? ProjectedTotalDeaths { get; set; }

        public double? MostRecentNewDeaths { get; set; }

  
    }
}

File: Covid.Web/ClientApp/src/app/model/states-data.ts
import { DatesData } from './dates-data';



export class StateData
{
    public selected: boolean | null;

    public stateAbbreviation: string;

    public stateName: string;

    public allDaysData: DatesData[];

    public dates: Date[];

    public mostRecentTotalCases: number | null;

    public projectedTotalCases: number | null;

    public mostRecentNewCases: number | null;

    public mostRecentTotalDeaths: number | null;

    public projectedTotalDeaths: number | null;

    public mostRecentNewDeaths: number | null;


}

